app/src/main/java/edu/temple/templetag/fragments/TagRecyclerViewFragment.java

```diff
@@ -59,7 +59,7 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container,
 
     public void updateDataSet(ArrayList<Tag> Tags){
         this.Tags = Tags;
-        tagRecyclerViewAdapter.clearDataSet(); // clear old data set of tags from past user location
+        //tagRecyclerViewAdapter.clearDataSet(); // clear old data set of tags from past user location
```

Comment:
Should be able to just remove this and then remove the clearDataSet() function all together from the TagRecyclerViewAdapter.